spec/models/api/v1/query_param_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Api::V1::QueryParam do
+  let(:dummy_controller) { Api::V1::BaseController.new.extend Api::V1::QueryParam }
```

Comment:
I'm still reusing `json_api_error` from base_controller. Could be an idea to extract all error handeling to a separate module, but felt a bit too much for now